wrappers/core18.go

```diff
@@ -155,6 +195,17 @@ func writeSnapdServicesOnCore(s *snap.Info, inter interacter) error {
 
 	// enable/start all the new services
 	for _, unit := range changed {
+		// systemd looks a the logical units, even if 'enabled' service
+		// symlink points to /lib/systemd/system location, dropping an
+		// identically named service in /etc overrides the other unit,
+		// therefore it is sufficient to enable the new units only
```

Comment:
This is fine for the "snapd" snap on core use-case. However when we move to a remodel things will be slightly more complicated. The core18 snap does not have enableed symlinks in /lib/systemd/system anymore. This means that in a remodel scenario we will need to create the etc symlinks. But given that this is for a different scenario it's fine to just add a FIXME here I think. We will eventually need to deal with it I'm afraid :/

Reply:
Could we somehow emply vendor presets here?